src/plugins/intel_gna/src/serial/gna_model_serial.cpp

```diff
@@ -269,7 +272,7 @@ void GNAModelSerial::Import(void* basePointer,
                 (model_header_.version.minor >= 3) ? readString(is) : std::string("input" + std::to_string(inputIndex));
             inputs[name] = InputDesc(name);
         }
-        if (model_header_.version.minor >= 5) {
+        if (model_header_.version.minor >= 5 && model_header_.version.minor <= 8) {
```

Comment:
does it mean that it should be not applied for 2.9?

Reply:
yes it does, I've added the explanation comments here.